### PR TITLE
Build and publish Docker images using Github

### PR DESCRIPTION
**Pull request by mvorisek**
Images from Docker Hub should be deleted completely as they are no longer updated.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,14 +4,15 @@ on:
...
+    - cron: '20 2 * * *'
 
 jobs:
   unit:
     name: Templating
+    if: github.event_name != 'push' || github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop'
     runs-on: ubuntu-latest
     container:
-      image: atk4/image
+      image: ghcr.io/mvorisek/image-php
```
Comment (romaninsh) [inline .github/workflows/ci.yml]: could we keep atk4/image here?

File: make.php [hunk 2]
```diff
@@ -111,92 +132,22 @@
...
+    - cron: \'20 2 * * *\'
 
 jobs:
   unit:
     name: Templating
+    if: github.event_name != \'push\' || github.ref == \'refs/heads/master\' || github.ref == \'refs/heads/develop\'
     runs-on: ubuntu-latest
     container:
-      image: atk4/image
+      image: ghcr.io/mvorisek/image-php
```
Comment (romaninsh) [inline make.php]: could we stick with atk4/ org owned repo please?

**Review by romaninsh — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mvorisek) [reply .github/workflows/ci.yml]: it will complicate merging, this is for build only

File: make.php (hunk 2 above)
Comment (mvorisek) [reply make.php]: done

**Comment by mvorisek** [comment]
@romaninsh feel free to adjust to push to Docker Hub instead of ghcr.io, secret should be put into the repo config

**Comment by mvorisek** [comment]
replaced by https://github.com/mvorisek/image-php

**Pull request closed without merging**